cmd/internal/kube/daemonset/daemonset.go

```diff
@@ -75,8 +79,40 @@ func (d *Daemonset) StartProcessInExistingPods() error {
 	return fmt.Errorf("not implemented")
 }
 
-func (d *Daemonset) KubernetesEventsWorker() {
-	// Not implemented
+func (d *Daemonset) KubernetesEventsWorker(done chan struct{}) {
```

Comment:
I think this should be a <-chan instead of bidirectional?  (Or did I get the arrow backwards?)